resources/assets/js/features/tripsList/components/ListWithTripsItems.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux';
+
+import { getTrips, filterPast,filterUpcoming } from "../actions";
+import TripsListItem from './TripsListItem';
+import './css/ListWithTripsItems.scss';
+
+class ListWithTripsItems extends  Component{
+
+    constructor(props){
+        super(props);
+        this.props.getTrips();
+        this.props.filterPast();
```

Comment:
make endpoint that will return past of future items instead of fetch all data and filter them